test/quantization/fx/test_numeric_suite_fx.py

```diff
@@ -2565,6 +2567,183 @@ def custom_convert_fn(module, to_print):
+    def test_add_loggers_linear_mod_fp32_fp32(self):
+        m = nn.Sequential(nn.Linear(2, 2), nn.Linear(2, 2))
+        example_input = (torch.randn(2, 2),)
+        qconfig_mapping = get_default_qconfig_mapping()
+        qconfig_mapping.set_module_name('0', None)
+        qconfig_mapping.set_module_name('1', None)
+        self._test_add_loggers_impl(m, example_input, qconfig_mapping)
+
+    @withQNNPACKBackend
+    def test_add_loggers_conv_bn_relu_fusion_quant(self):
+        m = nn.Sequential(nn.Conv2d(1, 1, 1), nn.BatchNorm2d(1), nn.ReLU())
+        m.eval()
+        example_input = (torch.randn(16, 1, 4, 4),)
+        qconfig_mapping = get_default_qconfig_mapping()
+        self._test_add_loggers_impl(m, example_input, qconfig_mapping)
+
+    @withQNNPACKBackend
+    def test_add_loggers_conv_bn_relu_fusion_fp32(self):
+        m = nn.Sequential(nn.Conv2d(1, 1, 1), nn.BatchNorm2d(1), nn.ReLU())
+        m.eval()
+        example_input = (torch.randn(16, 1, 4, 4),)
+        qconfig_mapping = get_default_qconfig_mapping()
+        qconfig_mapping.set_module_name('0', None)
+        qconfig_mapping.set_module_name('1', None)
+        qconfig_mapping.set_module_name('2', None)
+        self._test_add_loggers_impl(m, example_input, qconfig_mapping)
+
+    @withQNNPACKBackend
+    def qtest_add_loggers_functions(self):
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.w1 = nn.Parameter(torch.randn(2, 2))
+                self.b1 = nn.Parameter(torch.randn(2))
+                torch.nn.init.kaiming_uniform_(self.w1, a=math.sqrt(5))
+
+            def forward(self, x):
+                x = F.linear(x, self.w1, self.b1)
+                x = F.relu(x)
+                x = x + x
+                x = x + 1
+                # TODO(future PR): support first arg being a scalar
+                # x = 1 + x
+                x = torch.cat([x, x])
+                x = torch.cat([x, x])
+                x = torch.cat(tensors=[x, x])
+                # function not matchable by quantization
+                x = torch.nn.functional.rrelu(x)
+                x = F.linear(x, self.w1, self.b1)
+                return x
+
+        m = M().eval()
+        example_input = (torch.randn(16, 2),)
+        for qconfig_mapping in (
+            get_default_qconfig_mapping(),
+            QConfigMapping(),
+        ):
+            self._test_add_loggers_impl(m, example_input, qconfig_mapping)
+
+    @skip_if_no_torchvision
+    @withQNNPACKBackend
+    def test_add_loggers_mobilenet_v2(self):
+        import torchvision
+        m = torchvision.models.quantization.mobilenet_v2(
+            pretrained=False, quantize=False).eval()
+        example_input = (torch.randn(8, 3, 224, 224),)
+        qconfig_mapping = get_default_qconfig_mapping()
+        self._test_add_loggers_impl(m, example_input, qconfig_mapping)
+
+        if False:
+            # TODO(before land): remove this
```

Comment:
is this comment still valid?

Reply:
yes, this is just some extra checks which I'll remove before landing.  There isn't a good way to do these tests without referencing the old NS tools, so it doesn't make sense to add a test since the old tools will be deleted in the future.